src/certificate.rs

```diff
@@ -436,7 +436,7 @@ impl<'a> TbsCertificate<'a> {
     pub fn inhibit_anypolicy(
         &self,
     ) -> Result<Option<BasicExtension<&InhibitAnyPolicy>>, X509Error> {
-        self.get_extension_unique(&OID_X509_EXT_INHIBITANT_ANY_POLICY)?
+        self.get_extension_unique(&OID_X509_EXT_INHIBIT_ANY_POLICY)?
```

Comment:
I'm confused what happened here, I fixed this instance, and the one in `src/extensions/mod.rs` already in https://github.com/rusticata/x509-parser/commit/27dd7d2e7370737903251c15f554f887b05da827 no? Did that work get undone?

Reply:
Oh! I see now, this PR is targetting a `x509-parser-0.17` feature branch :bulb: 

In that case maybe next time it'd be better to cherry-pick the equivalent fixes from `master` where possible?

Reply:
Indeed, sorry I should have added it in the description.
I just created the `x509-parser-0.17` branch to backport fixes (like this one). We can of course also cherry-pick any change that is interesting for this branch.

This is also in anticipation of some future breaking changes when this crate will have to be updated to the next `asn1-rs` releases (which have a lot of pending breaking changes in its API, and will require adjusting `x509-parser`). Of course the goal is to make the update as easy as possible.